dbt_semantic_interfaces/implementations/element_config.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any, Dict
+
+from pydantic import Field
+from typing_extensions import override
+
+from dbt_semantic_interfaces.implementations.base import HashableBaseModel
+from dbt_semantic_interfaces.protocols.meta import PydanticSemanticLayerElementConfig
+from dbt_semantic_interfaces.protocols.protocol_hint import ProtocolHint
+
+
+class SemanticLayerElementConfig(HashableBaseModel, ProtocolHint[PydanticSemanticLayerElementConfig]):
```

Comment:
I think you got the names reversed here! This one should be pydantic, the other one is the protocol